crossbar/router/broker.py

```diff
@@ -261,7 +261,7 @@ def on_authorize_success(authorized):
                             for receiver in receivers:
                                 if me_also or receiver != session:
                                     # the receiving subscriber session might have been lost in the meantime ..
-                                    if receiver._transport:
+                                    if receiver._session_id and receiver._transport:
                                         receiver._transport.send(msg)
```

Comment:
I agree, would be cleaner. firing up pycharm and the code checking there spills out tons of private var access =( at some point, I'd like to clean that up ..